onnxruntime/core/providers/webnn/builders/impl/slice_op_builder.cc

```diff
@@ -66,18 +62,15 @@ Status SliceOpBuilder::AddToModelBuilderImpl(ModelBuilder& model_builder,
     std::string input_name;
     // This is an optional input, return empty vector.
     if (!is_required) {
-      if (input_defs.size() <= input_idx)
-        return Status::OK();
+      if (input_defs.size() <= input_idx) return Status::OK();
       input_name = input_defs[input_idx]->Name();
-      if (input_name.empty())
-        return Status::OK();
+      if (input_name.empty()) return Status::OK();
```

Comment:
Did autoformatting do this? I might be wrong, but I thought it generally put return statements on a separate line for increased branch point visibility?

Reply:
I used `clang-format` and it's formatted like this 🤔

Reply:
This is because @shiyi9801 set the `ColumnLimit` to 120.

Reply:
I changed the format back ✔